### PR TITLE
Implemented the Mobile view work, about, contact form sections

### PR DESCRIPTION
**Pull request by katfogy**
The Objectives was;
-To the mention session about using CSS flex box and grid
-Use `<ul>` and `<li>` to list items

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -41,6 +42,148 @@ <h3 class="connect">LET’S CONNECT</h3>
...
+                    <img src="assets/images/css.png" alt="">
+                    <h4>CSS</h4>
+                </li>
+            </ul>
+
+            <ul class="tools-container">
+                <li class="tools">framworks</li>
+                <li class="tools">skills</li>
+            </ul>
+        </article>
```
Comment (AbayomiOlaoye) [inline index.html:172]: - [ ] Good job with this attempt :clap: If you look closely, you will observe that these parts form part of the `About me` section where you show off your technical skill. Kindly nest it in within the `<section class="about-myself"></section>` tag. This makes your code tidier, and easily accessible to style, or in terms of accessibility to users with screen readers. :wink:

Comment (AbayomiOlaoye) [inline index.html:172]: - [ ] I must say that your first attempt is a blast! :fire: It's near perfect except that some minor details are not being carefully attended. Details like adequate spacing (`padding` and `margin`), `font-size`, and maybe some graphical contents. Kindly watch [this video](https://www.awesomescreenshot.com/video/15874312?key=ca22e8722654f7d4b0c59ebca15a82a7) that I have carefully recorded for you.

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
## STATUS: CHANGE REQUIRED ♻️ 

Hi @katfogy :wave:, 

Good job with this milestone so far!
There are just a few more issues to work on before moving on to the next project but you are almost there!

so_close

## Highlights
- Descriptive Pull Request message :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark: 
- Meaningful commit messages :heavy_check_mark: 
- No linter error :heavy_check_mark: 
- Descriptive ReadMe documentation :heavy_check_mark:
- Good UI implementation :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Zuhaib042 — APPROVED** [review]
Hi @katfogy 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Good Implementation of required changes

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**